code/drasil-lang/lib/Language/Drasil/Sentence/Fold.hs

```diff
@@ -126,3 +127,7 @@ numList s (y:z:xs)
 -- | Helper for numList that concatenates integers to strings.
 rangeSep :: Int -> Int -> String -> String
 rangeSep p q s = show p ++ s ++ show q
+
+-- | Helper to create default sentence for empty sections using NamedIdea
+emptySectSentence :: NamedIdea n => n -> Contents
+emptySectSentence var = foldlSP [S "There are no" +:+ plural var]
```

Comment:
I feel like this isn't the best place for this. Is there somewhere in `drasil-utils` or `drasil-doclang` that would be a better fit? This doesn't really "fold" a `Sentence`.

Reply:
Ok, thank you for clarifying! After I submitted the PR, I was also wondering if that belongs there. I will make that change right now!